src/globals/scss/vendor/@carbon/elements/scss/grid/12.scss

```diff
@@ -5,47 +5,33 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../layout/breakpoint';
-@import 'prefix';
-@import 'container';
-@import 'row';
-@import 'col';
+@import 'mixins';
 
-$breakpoints: map-merge(
+$carbon--12-column-grid: map-merge(
+  $carbon--grid-breakpoints,
   (
-    sm: (
-      columns: 12,
-    ),
-    md: (
-      columns: 12,
-    ),
-    lg: (
-      columns: 12,
-    ),
-    xlg: (
-      columns: 12,
-    ),
-    max: (
-      columns: 12,
-    ),
-  ),
-  $carbon--grid-breakpoints
+    lg:
```

Comment:
I'm a bit out of the loop on the grid changes, have the `sm` and `md` breakpoints moved?